html_reference.html

```diff
@@ -48,17 +49,17 @@ <h2 class="container">extra_curricular <i class="fas fa-biking" id="biking"></i>
 				</div>
 			</div>
 		</div>
-		<h3 class="container">portfolio  <i class="fas fa-palette" id="palette"></i></h2>
+		<h3 class="container">portfolio  <i class="fas fa-palette" id="palette"></i></h3>
 			<div class="porfolioButtonsContainer">
-				<button href="#" class="portfolioButtons" id="button1" type="button">PROJECT1</button>
-				<button href="#" class="portfolioButtons" id="button2" type="button">PROJECT2</button>
-				<button href="#" class="portfolioButtons" id="button3" type="button">PROJECT3</button>
-				<button href="#" class="portfolioButtons" id="button4" type="button">PROJECT4</button>
+				<a href="#" class="portfolioButtons" id="button1">PROJECT1</a>
```

Comment:
I assume that the ids on the buttons are required for the same reason as the classes fas and fa-palette?

I'd still recommend that you clear up why you've named them this way so that Ash knows for the final code review.

Reply:
Add a comment or something